src/PointwiseFunctions/AnalyticData/SelfForce/Scalar/CircularOrbit.cpp

```diff
@@ -0,0 +1,217 @@
+  const double r_plus = M * (1. + sqrt(1. - square(black_hole_spin_)));
+  const double r_0 = orbital_radius_;
+  const double r_star = gr::tortoise_radius_from_boyer_lindquist_minus_r_plus(
+      r_0 - r_plus, M, black_hole_spin_);
+  return tnsr::I<double, 2>{{{r_star, 0.}}};
+}
+
+// Background
+tuples::TaggedTuple<Tags::Alpha, Tags::Beta, Tags::Gamma>
+CircularOrbit::variables(
+    const tnsr::I<DataVector, 2>& x,
+    tmpl::list<Tags::Alpha, Tags::Beta, Tags::Gamma> /*meta*/) const {
+  const double a = black_hole_spin_ * black_hole_mass_;
+  const double M = black_hole_mass_;
+  const double r_plus = M * (1. + sqrt(1. - square(black_hole_spin_)));
+  const double r_minus = M * (1. - sqrt(1. - square(black_hole_spin_)));
+  const double r_0 = orbital_radius_;
+  const double omega = 1. / (a + sqrt(cube(r_0) / M));
+  const auto& r_star = get<0>(x);
+  const auto& cos_theta = get<1>(x);
+  const DataVector r_minus_r_plus =
+      gr::boyer_lindquist_radius_minus_r_plus_from_tortoise(r_star, M,
+                                                            black_hole_spin_);
+  const DataVector r = r_minus_r_plus + r_plus;
+  const DataVector delta = r_minus_r_plus * (r - r_minus);
+  const DataVector r_sq_plus_a_sq = square(r) + square(a);
+  const DataVector r_sq_plus_a_sq_sq = square(r_sq_plus_a_sq);
+  const DataVector sin_theta_squared = 1. - square(cos_theta);
+  const DataVector sigma_squared =
+      r_sq_plus_a_sq_sq - square(a) * delta * sin_theta_squared;
+  tuples::TaggedTuple<Tags::Alpha, Tags::Beta, Tags::Gamma> result{};
+  auto& alpha = get<Tags::Alpha>(result);
+  auto& beta = get<Tags::Beta>(result);
+  auto& gamma = get<Tags::Gamma>(result);
+  get(alpha) = delta / r_sq_plus_a_sq_sq;
+  const ComplexDataVector temp1 =
+      1. / r * std::complex<double>(0., 2. * a * m_mode_number_);
+  get(beta) = (-square(m_mode_number_ * omega) * sigma_squared +
+               4. * a * square(m_mode_number_) * omega * M * r +
+               delta * (square(m_mode_number_) / sin_theta_squared +
+                        2. * M / r * (1. - square(a) / M / r) + temp1)) /
+              r_sq_plus_a_sq_sq;
+  get<0>(gamma) =
+      -1. / r_sq_plus_a_sq * std::complex<double>(0., 2. * a * m_mode_number_) +
+      2. * square(a) * get(alpha) / r;
+  get<1>(gamma) = ComplexDataVector{cos_theta.size(), 0.};
+  get(alpha) *= sin_theta_squared;
+  return result;
+}
+
+// Initial guess
+tuples::TaggedTuple<Tags::MMode> CircularOrbit::variables(
+    const tnsr::I<DataVector, 2>& x, tmpl::list<Tags::MMode> /*meta*/) {
+  tuples::TaggedTuple<Tags::MMode> result{};
+  auto& field = get<Tags::MMode>(result);
+  get(field) = ComplexDataVector{get<0>(x).size(), 0.};
+  return result;
+}
+
+// Fixed sources
+tuples::TaggedTuple<
+    ::Tags::FixedSource<Tags::MMode>, Tags::SingularField,
+    ::Tags::deriv<Tags::SingularField, tmpl::size_t<2>, Frame::Inertial>,
+    Tags::BoyerLindquistRadius>
+CircularOrbit::variables(
```

Comment:
Can you document what this function is doing? Just from what's currently in the dox I have no way of verifying the equations.